osquery/tables/system/posix/augeas.cpp

```diff
@@ -15,13 +15,25 @@
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 
+namespace osquery {
+
+/**
+ * @brief Augeas lenses path.
+ *
+ * Directory that contains augeus lenses.
+ */
 #ifdef __APPLE__
-#define LENSES_PATH "/private/var/osquery/lenses"
+FLAG(string,
```

Comment:
Ok. split the commit into separate pull requests. Let me know if there is a way to work-around above warnings. Thanks